src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,114 @@
+import cn from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { FC } from 'react';
+import { PersonLink } from '../PersonLink';
+import { SearchLink } from '../SearchLink';
+import { Person } from '../../types';
+import { SearchFilterParams } from '../../types/SearchFilterParams';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get(SearchFilterParams.Sort)
+  || SearchFilterParams.None;
+  const order = searchParams.get(SearchFilterParams.Order)
+  || SearchFilterParams.None;
+
+  const getLinkClass = (param: string) => cn(
+    'fas',
+    { 'fa-sort': param !== sort },
+    { 'fa-sort-up': sort && sort === param && !order },
+    { 'fa-sort-down': sort && sort === param && order === 'desc' },
+  );
+
+  const getLinkParams = (
+    param: string,
+  ):
+  | { sort: string }
+  | {
+    sort: null;
+    order: null;
+  }
+  | { order: string } => {
```

Comment:
This type annotation looks a bit confusing, you can create a type where these fields are optional or can be string/null
```typescript
interface LinkParams {
  sort?: string | null;
  order?: string | null;
}
```